styles/lazy-styles.css

```diff
@@ -25,3 +25,11 @@
     font-style: normal;
     font-display: swap;
 }
+
+@font-face {
+    font-family: VltIcons;
+    src: url('/fonts/VltIcons.ttf') format('truetype');
```

Comment:
Clients icon font that includes an upper right pointing arrow to use as a decoration on the 'For Home' link in the upper nav menu.